### ISSUE TITLE
Publishing Error: The table 'Time Sheet Line' cannot be located.

### ISSUE THREAD

**Issue by teganZT** [hougaard/Simple-Object-Designer#97]
Hi Erik,

I am getting the attached error file any time I try to deploy in my environment. With a deployment failure I would usually just undo the field I had just added in order to fix it. The problem is, this error was caused by someone else in my org and we're not sure what change they made to trigger the error. Here is the full error message:

> Deployment failed. Errors: App ID : 9ce6b598-1f62-47f8-b4a4-c509434189b4 Message : { Could not upgrade the NAV extension 'New Fields by WDierks 1.0.0.65' due to the following error: 'TableExtension 50116 SODTime_Sheet_Line :: The table 'Time Sheet Line' cannot be located. Removing tables is not allowed unless they are temporary or are being moved by migration to another app.'. }

Do you have any advice or documentation on how I can read error messages to determine what is causing them. I removed all new or existing fields that were added to or from Time Sheet Line in SOD, but the error persists.

Thank you in advance for the help.
[Result{4F874908-FD56-47E1-9D15-A823E7A76C3E}.txt](https://github.com/hougaard/Simple-Object-Designer/files/9311422/Result.4F874908-FD56-47E1-9D15-A823E7A76C3E.txt)

**Comment by hougaard** [comment]
If you need to remove a field, you must upload the generated .app file manually to Extension Management, with Schema Update Mode set to Force.

**Comment by teganZT** [comment]
Hi Erik,

I still don't fully understand. I just want to determine what change in the SOD was made and undo it. The error message mentions the Time Sheet Line table, but we do not want to delete that table. Or, is there a way to revert to a previous .app version before my colleague made the change?

**Comment by teganZT** [comment]
I was able to find the change I made and reversed it. In the future we'll be more careful about undoing changes immediately after getting a failure to ensure they don't stay in place. 

One thing that might make it easier for less-Dynamics knowledgeable users like myself would be a "Undo last change" option after a publishing error. Without that it takes a long time to scroll though all the possible fields that were placed on pages or reports which could be causing the error. 

Thank you again for the help
